README.md

```diff
@@ -249,7 +257,200 @@ AdyenCardComponent(
   },
 );
 ``` 
+<br>
+
+### Apple Pay Component with Sessions flow
+1. From your backend, make a [`sessions`](https://docs.adyen.com/api-explorer/Checkout/71/post/sessions) request. The response contains:
+- `sessionData`: the payment session data.
+- `id`: a unique identifier for the session data.
+- The request body.
+
+Pass these values to your app.
+
+2. In your Flutter app, create a `ApplePayComponentConfiguration` instance. It requires a `ApplePayConfiguration` which contains your merchant id and merchant name.
```

Comment:
Would recommend to put this as a separate step as well, since they explicitly have to create another object. 

2. In your Flutter app, create two objects (?)
    1.  `ApplePayConfiguration` object: includes your merchant id and merchant name. 
    2.  `ApplePayComponentConfiguration`: includes the component configuration, and requires the `applePayConfiguration` object you created. 
4.